src/contracts/test/NonPayable.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+contract NonPayable {
+    // solhint-disable-next-line no-empty-blocks
+    fallback() external {}
```

Comment:
I suppose the warning here is acceptable. Is there anyway to suppress it?

```
When fallback is not payable you will not be able to receive ether
```

Reply:
Nice! I just missed this.